client/my-sites/checkout/src/components/checkout-main.tsx

```diff
@@ -23,6 +23,7 @@ import { errorNotice, infoNotice } from 'calypso/state/notices/actions';
 import isPrivateSite from 'calypso/state/selectors/is-private-site';
 import isAtomicSite from 'calypso/state/selectors/is-site-automated-transfer';
 import { isJetpackSite } from 'calypso/state/sites/selectors';
+import { useCheckoutMigrationIntroductoryOfferSticker } from '../../../../data/site-migration/use-checkout-migration-introductory-offer-sticker';
```

Comment:
```suggestion
import { useCheckoutMigrationIntroductoryOfferSticker } from 'calypso/data/site-migration/use-checkout-migration-introductory-offer-sticker';
```

This is a bit easier to read. WDYT?